envpool/box2d/__init__.py

```diff
@@ -26,24 +26,28 @@
   _LunarLanderDiscreteEnvSpec,
 )
 
-CarRacingEnvSpec, CarRacingDMEnvPool, CarRacingGymEnvPool = py_env(
-  _CarRacingEnvSpec, _CarRacingEnvPool
-)
+(
```

Comment:
@Trinkle23897 Would it be better that we can make this less verbose? e.g. with a helper function like
```
def register_to_py(cpp_shared_lib):
  dict_of_name_to_env = py_env(cpp_shared_lib)
  globals().update(dict_of_name_to_env)
  __all__ = dict_of_name_to_env.keys()
```

Reply:
I mean, we could make this change as a separate PR after this one.

Reply:
Sounds good to me